### PR TITLE
Chore: Bump CNI/ CNS version from v1.6.0 to v1.6.3.

### PR DESCRIPTION
**Pull request by sivakami-projects**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:

```
none
```

### CODE REVIEW

File: parts/linux/cloud-init/artifacts/components.json [hunk 1]
```diff
@@ -44,7 +44,8 @@
       "multiArchVersions": [
         "v1.5.28",
```
Comment (rbtr) [inline parts/linux/cloud-init/artifacts/components.json:45]: let's pull `v1.5.28`, `v1.5.32` should be fully rolled out by the time this VHD goes to AKS

File: parts/linux/cloud-init/artifacts/components.json [hunk 2]
```diff
@@ -74,7 +81,8 @@
         "v1.4.52",
         "v1.5.26",
```
Comment (rbtr) [inline parts/linux/cloud-init/artifacts/components.json:82]: same for `v1.5.26`

File: parts/linux/cloud-init/artifacts/components.json [hunk 3]
```diff
@@ -475,7 +489,9 @@
             "versions": [
               "1.4.54",
               "1.5.28",
```
Comment (rbtr) [inline parts/linux/cloud-init/artifacts/components.json:491]: same

**Review by rbtr — CHANGES_REQUESTED** [review]
cc @QxBytes

File: parts/linux/cloud-init/artifacts/components.json (hunk 1 above)
Comment (rbtr) [reply parts/linux/cloud-init/artifacts/components.json:45]: + @AlisonB319 FYI

Comment (cameronmeissner) [reply parts/linux/cloud-init/artifacts/components.json]: I'm fine either way, seems like this PR addresses that now

**Review by rbtr — APPROVED** [review]

**Review by cameronmeissner — APPROVED** [review]

**Comment by QxBytes** [comment]
Currently the 1.5.32 cni/cns version is still rolling out and so if we remove the 1.5.28 cni/cns version existing clusters/clusters in regions which haven't been upgraded may be affected.

**Review by QxBytes — APPROVED** [review]
VHD changes will likely take 1-2 weeks before reaching aks rp, so removing v1.5.28 while v1.5.32 is currently rolling out should be fine.

**Pull request merged**